src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheck.java

```diff
@@ -66,6 +66,25 @@
  * r = 5; int t; //violation here
  * </pre>
  * <p>
+ * Note: resource declarations can contain variable definitions
+ * and variable references (from java9).
+ * This check is only applied to variable definitions.
```

Comment:
should be `This Check`

@rnveach , please confirm.

Reply:
Grammatically speaking it should be lowercase `c`.
In terms of other documentations, I don't remember us doing uppercase `Check`.
Some examples we have are like:
`To configure the check:`
`To configure the check for an @author tag:`
`Currently the check doesn't permit horizontal alignment`
Some areas worked around grammar by making `check` the first word of the sentence.
`Checks that ...`

Reply:
Few lines below it is capital - https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/OneStatementPerLineCheck.java#L69

For other check it is lower-case:
https://github.com/checkstyle/checkstyle/blob/85660fa4cb60fec871b1e5343694236e314da92f/src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/LineLengthCheck.java#L51

I think it can be lowercase here

Reply:
@romani other comments are corrected, please take a look and let me know if I need to correct this (make c capital) as well

Reply:
Sorry for confusion, let's keep it lowercase. For some reason I thought it make it more distinguished from verb, as it is special term in our project.